stat/distuv/bernoulli.go

```diff
@@ -83,15 +80,22 @@ func (Bernoulli) NumParameters() int {
 
 // Prob computes the value of the probability distribution at x.
 func (b Bernoulli) Prob(x float64) float64 {
-	return math.Exp(b.LogProb(x))
+	if x == 0 {
+		return 1 - b.P
+	}
+	if x == 1 {
+		return b.P
+	}
+	return 0.
 }
 
-// Quantile returns the inverse of the cumulative probability distribution.
+// Quantile returns the minimum value of x from amongst all those values whose CDF value exceeds or equals p.
```

Comment:
Please revert this.

Reply:
I can, but can I ask why? The CDF of a Bernoulli distribution does not really have an inverse. The new comment is more precise mathematically.

Reply:
Yeah, OK.